definitions/contested/json/AuthorisationCaseState/AuthorisationCaseState.json

```diff
@@ -501,7 +515,14 @@
     "CaseTypeID": "FinancialRemedyContested",
     "CaseStateID": "consentedOrderSubmitted",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
-    "CRUD": "R"
+    "CRUD": "CRUD"
```

Comment:
this change is to be reverted before testing, need to ensure that the new ''Awaiting Judiciary Response (consent)' state has CRUD for Judge when https://tools.hmcts.net/jira/browse/FR-1746 is merged